src/objects/drawCurve/index.js

```diff
@@ -29,15 +29,17 @@ export default function drawCurve(a, b) {
 
   const lineGeometry = new THREE.BufferGeometry();
   const positions = new Float32Array(points.length * 3);
-  for (let ii = 0; ii < points.length; ii += 1) {
-    positions[ii * 3 + 0] = points[ii].x;
-    positions[ii * 3 + 1] = points[ii].y;
-    positions[ii * 3 + 2] = points[ii].z;
+  for (let i = 0; i < points.length; i += 1) {
```

Comment:
I figured `i` was better than `ii` here, as there is no `i` used in this method 🤷 All changes to this file are completely semantics / formatting and do not change behavior in any way.

Happy to revert these changes if you'd prefer. 🐒

Reply:
Nah, that's fine. `ii` was from when this was all in one GIANT file.